docs/reference/workflow-reference.md

```diff
@@ -1,3 +1,491 @@
 # Workflow
 
-::: up42.workflow.Workflow
+The Workflow class enables access to the UP42 [analytics functionality](analytics.md).
+
+A workflow is a sequence of data blocks and processing blocks. It defines an order of operations that start with a data block, which may be followed by up to five processing blocks.
+
+```python
+workflow = up42.initialize_workflow(
+    project_id="68567134-27ad-7bd7-4b65-d61adb11fc78",
+    workflow_id="7fb2ec8a-45be-41ad-a50f-98ba6b528b98",
+)
+```
+
+## Projects
+
+### max_concurrent_jobs
+
+The `max_concurrent_jobs` attribute returns the maximum number of jobs that can run simultaneously.
+
+The returned format is `int`.
+
+<h5> Example </h5>
+
+```python
+workflow.max_concurrent_jobs
+```
```

Comment:
Not needed

Reply:
What I understand about this changes is that we are moving from automatic documentation to md documents to have better control of the format and styling. Also, more consistency with the general documentation. @dustgalactic is that right? . In that case, Does 'Not needed' mean that we should remove the section, or keep the automatic documentation? Again, this type of feedback requires more context in my opinion.